package.json

```diff
@@ -15,6 +15,8 @@
   "scripts": {
     "build": "yarn workspace frontend build",
     "deploy-vcr": "yarn build && vcr deploy",
+    "deploy-vcr-local": "dotenv -- yarn deploy-vcr-local-with-env",
+    "deploy-vcr-local-with-env": "vcr deploy -f ./vcr_local.yml --app-id $VCR_APP_ID",
```

Comment:
to use the dotenv values you need to use a separate command. If I put all this in one line $VCR_APP_ID would not be defined.

Reply:
So this means that if you want to deploy the app with the variables in your env file you need to run 
`yarn deploy-vcr-local`
correct?

Reply:
Yes, the other way is to put this in a string in a bash command. 
Maybe we could rename this to something less user friendly